src/utilops.cc

```diff
@@ -406,6 +409,17 @@ struct UtilityData {
 	void (*details_func)(UtilityData *ud, FileData *fd);
 	gboolean (*finalize_func)(FileData *fd);
 	gboolean (*discard_func)(FileData *fd);
+
+	/* progress dialog */
```

Comment:
`UtilityData` is already huge. Is it possible to split progress dialog data to separate class/struct and use member of that type here?